File: 11721.c
#include<stdio.h>

int main(){
    char input[100];                //index 0 부터 체우면 마지막 null까지 해서 100 맞음
    scanf("%s", input);            //배열은 그 자체로 주소이므로 &을 붙이지 않아야함.
    
    for(int i = 0; i < strlen(input); i++){    //strlen으로 문자열 길이 알 수 있음.
        printf("%c", input[i]);
        if(i % 10 == 9)    printf("\n"); 
    }
    return 0;
}

File: 11721.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String input = sc.next();
        sc.close();
        
        for(int i = 0; i < input.length(); i++){            //C에선 strlen(), java에선 .length()
            System.out.print(input.charAt(i));                //C에선 char 배열의 index로 생각, java에서는 .charAt()
            if(i % 10 == 9)    System.out.println();
        }
    }
}
